integration/testdata/nodejs_simple_app/plan.toml

```diff
@@ -0,0 +1,11 @@
+[[requires]]
```

Comment:
Do we still need to have a plan.toml since we are now using the Node.js buildpack and extension? Could we just not build the application with the builder that was generated and hav it figure out the build plan?

Reply:
Good point, using only the node-engine buildpack and the extension the build fails ` [detector] fail: paketo-community/ubi-nodejs-extension@0.0.1 provides unused node` Although i figured out that, by using build plan buildpack with this plant.toml
```
[[requires]]
  name = "node"

  [requires.metadata]
    launch = true
```
and the extension the app builds and runs. Which is godd as we removing npm and node engine buildpacks.

I Believe we can also exclude buildplan by changing what extension provides on the detect phase. That way we will only have extension being used during build.

Reply:
For now, an improved fix can be removing the node-engine and npm-install buildpack in conjuction with the aforemention plan.toml

Reply:
fix done.